tests/tt_metal/tt_fabric/fabric_data_movement/test_basic_1d_fabric.cpp

```diff
@@ -122,7 +122,7 @@ void get_mcast_receivers(
             auto curr_fabric_node_id = node;
             for (uint32_t i = 0; i < mcast_ref[trunk_direction].size(); i++) {
                 auto neighbors = control_plane.get_intra_chip_neighbors(curr_fabric_node_id, trunk_direction);
-                if (neighbors.size() > 0) {
+                if (!neighbors.empty()) {
```

Comment:
Potential follow-up to this PR: early-return:
```suggestion
                if (neighbors.empty()) {
                    continue;
                }
                // de-dent following logic
```